### DISCUSSION TITLE
Does anyone know the syntax to run all CIS checks against both M365 and Azure?

### DISCUSSION THREAD

**Discussion by CaptJoeKPR** [silverhack/monkey365#45] [Q&A]
One of their screenshots seems to show both M365 and Azure being run on the same report.  I can't seem to figure out the syntax however

**Comment by silverhack** [comment]
Hi @CaptJoeKPR, 

There is no way to run all checks on both Azure and M365. When scanning with Monkey365, you should target Azure or Microsoft365 separately.

Cheers,
